BFit/Controller/ProfileViewController.swift

```diff
@@ -60,19 +60,29 @@ class ProfileViewController: UIViewController, UIImagePickerControllerDelegate,
         }
     }
     
-    
     func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        let imageURL = info[UIImagePickerController.InfoKey.imageURL] as! URL
         let pickedImage = info[UIImagePickerController.InfoKey.originalImage] as? UIImage
         profilePic.image = pickedImage
+        postImage(imgURL: imageURL as URL)
         dismiss(animated: true, completion: nil)
     }
     
+    func postImage(imgURL : URL) {
+        cloudinary.createUploader().upload(url: imgURL, uploadPreset: "rgflevhw")
+            .response({ (response, error) in
+                if let result = response {
+                    print(result.url!)
+                } else if (error != nil) {
+                    print("error:", error as Any)
```

Comment:
Another thing I considered while we're waiting for endpoints is that we could work on setting up some better error handling, I think there's a number of places where we're still just printing the error for now

Reply:
very true, leveraging the errors may very well be useful to the user and in turn our debugging based on user feedback